### ISSUE TITLE
vue 2.3+ v-bind support  modifier async

### ISSUE THREAD

**Issue by stevieyu** [vuejs/eslint-plugin-vue#527]
**Tell us about your environment**

* **ESLint Version:** 4
* **eslint-plugin-vue Version:** 3.7
* **Node Version:** 10

**Please show your full configuration:**
notice: 'v-bind' directives don't support the modifier 'async'
but vue support .sync
link: https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier

```




```

**What did you do? Please include the actual source code causing the issue.**

```vue





```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**

**Comment by michalsnik** [comment]
Hey @stevieyu ! The `sync` modifier is being supported: https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/rules/valid-v-bind.js#L18

I tested it in a second, and I can't see any errors. Please try newer version of `eslint-plugin-vue`, current one is `4.7.0`. If that doesn't help - feel free to reopen the issue and update code examples, as currently I can't see any code above ☝️

**Comment by hyedong** [comment]
?
